tidb.go

```diff
@@ -243,9 +243,24 @@ func NewStore(uri string) (kv.Storage, error) {
 
 var queryStmtTable = []string{"explain", "select", "show", "execute", "describe", "desc"}
 
+func trimSQL(sql string) string {
```

Comment:
We need to strip comments start with --
see https://github.com/pingcap/tidb/issues/199